src/main/kotlin/com/exactpro/th2/codec/EncodeProcessor.kt

```diff
@@ -99,6 +99,9 @@ class EncodeProcessor(
         }
     }
 
-
-
+    private fun sendErrorEvents(errorMsg: String, parentEventIds: Set<String>, msgGroup: MessageGroup,
+                                cause: Throwable, additionalBody: List<String>?){
```

Comment:
I think it is better to use a non-nullable empty collection to indicate that there is no additional data